Presentation.Web/App_Start/NinjectWebCommon.cs

```diff
@@ -23,9 +30,17 @@ public static void Start()
             DynamicModuleUtility.RegisterModule(typeof(NinjectHttpModule));
 
             //Setup for web application
-            bootstrapper.Initialize(()=> new KernelBuilder().ForWebApplication().Build());
+            bootstrapper.Initialize(() =>
+            {
+                var kernel = new KernelBuilder().ForWebApplication().Build();
 
-            //Setup for hangfire with a different setup
+                //Only register the interceptor once per application (Hangfire job is the same application scope)
+                DbInterception.Add(new EFEntityInterceptor(() => kernel.Get<IOperationClock>(), () => kernel.Get<Maybe<ActiveUserIdContext>>(), () => kernel.Get<IFallbackUserResolver>()));
```

Comment:
Ahh så på denne måde får du aktiveret ninject kernel og resolvet de ting der måtte være der og så derefter tilføjet interceptoren. Så er jeg glad for jeg smed denne opgave tilbage da de løsninger jeg var ved at overveje var noget forsøg med lazy loading

Reply:
Ja det der interception er ikke integreret i almindelig pipeline, så vi er bare heldige at Ninject kan finde den rette context når den resolver i kontekst af hhv. request og/eller hangfire tråd